intelligence-services/latest/config/index.md

```diff
@@ -111,14 +111,15 @@ A number of environment variables allow you to specify the configuration options
 4. Add the following configuration to override the settings for Digital Workspace in your deployment.
 
     ```yaml
-  digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:2.4.0-adw
-    environment:
-      BASE_PATH: ./
-      APP_CONFIG_PLUGIN_AI_SERVICE: "true"
-    volumes:
-      - ./ai-view-extension.json:/usr/share/nginx/html/assets/plugins/ai-view-extension.json
+    digital-workspace:
+        image: quay.io/alfresco/alfresco-digital-workspace:2.4.2
```

Comment:
This version looks very old. It is from 7 months ago. The https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/docker-compose.yml#L149 currently has the ACS 7.2.0 compatible values and is 2.7.0. There is also now a 2.8.0, but I'm not the right person to know what version should be here.

There is a second reference to 2.4.2 in this PR, if you wish to change this.

Reply:
Switched to ADW 2.8